test/src/SpendPermissions/isAuthorized.t.sol

```diff
@@ -0,0 +1,16 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.23;
+
+import {Test, console2} from "forge-std/Test.sol";
+
+import {SpendPermissions} from "../../../src/SpendPermissions.sol";
+
+import {SpendPermissionsBase} from "../../base/SpendPermissionsBase.sol";
+
+contract IsAuthorizedTest is Test, SpendPermissionsBase {
+    function setUp() public {
+        _initializeSpendPermissions();
+    }
+    function test_isAuthorized_success_true() public {}
+    function test_isAuthorized_success_false() public {}
```

Comment:
we generally want a test for every branch of logic, not just outcome. I think it will become more obvious which case is missing if you reword `true/false` to describe what preconditions are met to form that outcome. Hint, there are two cases where `isAuthorized` returns `false`